x-pack/plugins/secops/public/graphql/types.ts

```diff
@@ -224,7 +224,7 @@ export interface EcsEdges {
 }
 
 export interface Ecs {
-  _id?: string | null;
+  _id: string;
```

Comment:
Thanks for addressing https://github.com/elastic/ingest-dev/issues/295 by making this mandatory, per our conversation with @spong !